ui_framework/src/components/vertical_rhythm/_vertical_rhythm.scss

```diff
@@ -0,0 +1,23 @@
+.kuiVerticalRhythm {
```

Comment:
What do these mixins do that the variables themselves don't already provide? When would we ever swap the content inside of them, versus just changing the value of the variable?

As a react component when is it ever a good idea to use `<KuiVerticalRhythm size="whatever">` versus just passing it onto the element?

If we did use something like this, it should be called `BottomMargin` I think? Vertical Rhythm is most often associated with a holistic system to space _bodies_ of text. It's usually a mix of lots of different values, not just spacing, but size, line-height...etc.